mbedtls/tests/save_restore.rs

```diff
@@ -9,10 +9,12 @@
 use mbedtls::cipher;
 use mbedtls::cipher::raw::{CipherId, CipherMode, CipherPadding};
 use mbedtls::cipher::{Cipher, Decryption, Encryption, Fresh, Authenticated, Traditional};
+#[cfg(buggy)]
 use serde_cbor::{de, ser};
 
 const ZERO_16B: &'static [u8] = &[0u8; 16];
 
+#[cfg(buggy)]
```

Comment:
Did we want to have a follow on issue or similar to address these disabled tests?

Reply:
I don't think it's necessary: we don't use this crate for AES at all, and I'd like to stop using it entirely once there's an audited, no-std capable, X509+RSA crate in pure Rust.